scripts/test-start-local-ip.sh

```
@@ -0,0 +1,63 @@
+            depth += opens - closes
+            if (depth == 0) {
+                exit
+            }
+        }
+    ' "$START_SH"
+}
+
+printf '%s\n' '#!/usr/bin/env bash' 'printf "%s\n" "   interface: utun5"' > "$FAKE_BIN/route"
+printf '%s\n' \
+    '#!/usr/bin/env bash' \
+    'case "${1:-}" in' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)